Automatic Validation ended with: 
> 2026-04-15T20:26:03.0531741Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]
2026-04-15T20:26:03.0532908Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]

(Automated response - build 1248.)

--- a/manifests/m/MesterPerfect/IlmQuiz/1.0.0/MesterPerfect.IlmQuiz.locale.en-US.yaml
+++ b/manifests/m/MesterPerfect/IlmQuiz/1.0.0/MesterPerfect.IlmQuiz.locale.en-US.yaml
@@ -31,8 +31,5 @@ Tags:
 ReleaseNotes: Initial release of IlmQuiz. Features an interactive Islamic quiz experience with cross-platform support and strong accessibility for blind and visually impaired users.
 ReleaseNotesUrl: https://github.com/MesterPerfect/IlmQuiz/releases/tag/v1.0.0
 InstallationNotes: Thank you for installing IlmQuiz! You can find more useful projects and resources on my GitHub profile.
-Icons:
-- IconUrl: https://raw.githubusercontent.com/MesterPerfect/IlmQuiz/master/assets/icons/app_icon.png
-  IconFileType: png
 ManifestType: defaultLocale
 ManifestVersion: 1.12.0

